commands/create.rb

```diff
@@ -0,0 +1,42 @@
+class Command
+  def self.create
+    puts 'Example package names: libnvme, crew-profile-base'.lightblue
+    name = gets('Enter the package name:').chomp.capitalize.tr('-', '_')
```

Comment:
Check for invalid package names?   First character must be a letter, only alphanumeric and - or _ allowed, search packages for duplicate names, etc.